documentation/DOCUMENTATION.md

```diff
@@ -0,0 +1,851 @@
+   and automatically selects the first process in the list.
+
+   > **Note** If you previously profiled your application and saved a capture,
+   > you can click **Load a capture** to directly load the capture from this
+   > window. For ease of access, Orbit maintains a list of captures that you
+   > saved recently.
+
+3. To confirm the selected process and continue to the main window, click
+   **Start Session** or <kbd>Double-Click</kbd> the process. In the
+   **Main Window**, you can see the following:
+
+   * The selected process and instance in green.
+   * The **Modules** section in the **Symbols** tab displays all the loaded
+   modules for the process, including your application binary and all libraries
+   that your application uses. Orbit attempts to load symbols for all modules
+   automatically. You can see the state of symbol loading in the first column of
+   the **Modules** section.
+
+   ![Orbit's main window with processes and modules][orbit_main_window_startup]
+
+4. To return to the startup window and select a different process, or connect
+   to a different instance, click **End Session**.
+
+   > **Note** If you loaded a capture from file, the right side of the menu bar
+   displays the filename of the capture instead of connection details.
+
+### Record a capture with callstack sampling
+
+This section shows you how to record a basic capture that samples callstacks in
+your application.
+
+1. On the left-hand side in the **Capture** tab, click
+   <img src="../icons/outline_play_arrow_white_48dp.png" width="18" height="18" alt="Play icon" />
+   to start taking a capture. After a few seconds, click
+   <img src="../icons/outline_stop_white_48dp.png" width="18" height="18" alt="Stop icon" />
+   to stop capturing. The capture looks like the screenshot below:
+
+   ![Orbit default capture][orbit_main_window_default_capture]
+
+   The **Sampling** tab automatically shows a report of the collected samples
+   anytime a capture is stopped.
+
+1. Click a row to select a function. You can inspect sampled callstacks that
+   involve this function at the bottom.
+
+The **Capture Options** dialog (click
+<img src="../icons/outline_settings_white_48dp.png" width="18" height="18" alt="Settings icon" />)
+lets you toggle between two methods of callstack unwinding: **DWARF**, the
+default method, and **Frame pointers**. The **Frame pointers** method has much
+less overhead and produces the same results as the **DWARF** method.
+
+However, to unwind using frame pointers, you must compile *all* target binaries
+(application and all libraries) with the `‑fno‑omit‑frame‑pointer` compiler flag.
+Orbit still supports leaf functions (functions without a call instruction) that
+do not set the frame pointer, which means it also supports binaries compiled
+with `‑momit‑leaf‑frame‑pointer` (as long as `‑fno‑omit‑frame‑pointer` is used).
+All application surface libraries that are part of a Stadia instance are already
```

Comment:
Do we want to keep stuff specific to Stadia?

Reply:
Good catch. I simply removed the sentence. I don't think it makes sense to keep it.